pkg/pot/ast.go

```diff
@@ -0,0 +1,143 @@
+package pot
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"golang.org/x/text/language"
+)
+
+type HeaderNode struct {
+	Language    language.Tag
+	Translator  string
+	PluralForms PluralForm
+}
+
+type PluralForm struct {
+	Plural   string
+	NPlurals int
+}
+
+type MessageNode struct {
+	MsgCtxt               string
+	MsgId                 string
+	MsgIdPlural           string
+	TranslatorComment     string
+	ExtractedComment      string
+	Reference             string
+	Flag                  string
+	MsgCtxtPrevCtxt       string
+	MsgIdPrevUntPluralStr string
+	MsgIdPrevUnt          string
+	MsgStr                []string
+}
+
+type Po struct {
+	Header   *HeaderNode
+	Messages []*MessageNode
+}
```

Comment:
No need for pointers.

```suggestion
type Po struct {
	Header   HeaderNode
	Messages []MessageNode
}
```